src/main/osd/osd.c

```diff
@@ -90,6 +91,9 @@
 #include "hardware_revision.h"
 #endif
 
+#define VIDEO_BUFFER_CHARS_PAL    480
+#define IS_DISPLAY_PAL (displayScreenSize(osdDisplayPort) == VIDEO_BUFFER_CHARS_PAL)
```

Comment:
There seems to be little value in using a define for this.

Reply:
In INav there are more places where PAL/NTSC check is made (e.g. crosshairs, horizon sidebars).
Maybe we'll need it there as well.

I prefer a define, as its implementation is not very elegant currently...